<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorPosition</class>
 <widget class="QWidget" name="MotorPosition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSpinBox,QDoubleSpinBox {
	color: rgb(0,0,127);
	border: 1px solid rgb(0,0,127);
	border-radius: 2px;
}

QSpinBox:hover,QDoubleSpinBox:hover {
	background-color: rgba(0, 0, 0, 16);
}

QSpinBox::up-button,QDoubleSpinBox::up-button,
QSpinBox::down-button,QDoubleSpinBox::down-button {
	border-left: 1px solid rgb(0,0,127);
	background-color: transparent;
}

QSpinBox::up-button:hover,
QDoubleSpinBox::up-button:hover,
QSpinBox::down-button:hover,
QDoubleSpinBox::down-button:hover
{
	border-left: 1px solid rgb(0,0,127);
	background-color: rgba(0, 0, 0, 32);
}

QSpinBox::up-arrow,QDoubleSpinBox::up-arrow{
	image: url(:/icons/arrowup.png);
	width: 5px;
}

QSpinBox::down-arrow,QDoubleSpinBox::down-arrow{
	image: url(:/icons/arrowdown.png);
	width: 5px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tau:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Damping:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Proportional Recovery Velocity:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Constant Recovery Velocity:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="spinBoxTau">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="spinBoxDamping">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="spinBoxPV">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="spinBoxCV">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
